pkg/scheduler/apis/config/types.go

```diff
@@ -138,6 +138,9 @@ type KubeSchedulerProfile struct {
 // Enabled plugins are called in the order specified here, after default plugins. If they need to
 // be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
 type Plugins struct {
+	// PreEnqueue is a list of plugins that should be invoked before adding pods to the scheduling queue.
```

Comment:
Need to specify which queue of the scheduling queue to enter.

```suggestion
	// PreEnqueue is a list of plugins that should be invoked before adding pods to the activeQ in the scheduling queue.
```

Reply:
probably not worthing indicating the details in the comments. (the comments of QueueSort also says "in the scheduling queue")